datafusion/sql/Cargo.toml

```diff
@@ -47,6 +47,7 @@ arrow-schema = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
 log = { workspace = true }
+regex = { version = "1.8" }
```

Comment:
I think we need to move regex to top level, as it is used in much of packages. It can be done as followup

Reply:
I moved it in 32aa0e9.